### ISSUE TITLE
OpenSSL Unable to Open Config

### ISSUE THREAD

**Issue by anoadragon453** [termux/termux-packages#2865]
Trying to generate a certificate request with openssl provided by the `openssl-tools` packages yields the following error as no openssl.cnf file has been provided with the package:

```
$ openssl genrsa 4096 > domain.key
$ openssl req -new -sha256 -key domain.key -subj "/CN=example.com" > domain.csr
Can't open /data/data/com.termux/files/usr/etc/tls/openssl.cnf for reading, No such file or directory
518150247752:error:02001002:system library:fopen:No such file or directory:crypto/bio/bss_file.c:72:fopen('/data/data/com.termux/files/usr/etc/tls/openssl.cnf','r')       518150247752:error:2006D080:BIO routines:BIO_new_file:no
such file:crypto/bio/bss_file.c:79:                      unable to find 'distinguished_name' in config            problems making Certificate Request                      518150247752:error:0E06D06A:configuration file routines:N
CONF_get_string:no conf or environment variable:crypto/conf/conf_lib.c:270:
```

**Comment by xeffyr** [comment]
Seems that `openssl.cnf` is not getting packaged at all.

**Comment by anoadragon453** [comment]
Would be nice if it was :)

**Comment by anoadragon453** [comment]
Thanks all, looking forward to giving it a shot!

**Comment by fornwall** [comment]
Thanks for reporting! This is now fixed in the `1.1.1-2` version of the package, run `pkg up` to update!
